coriolis_openstack_utils/actions/network_actions.py

```diff
@@ -382,6 +383,15 @@ def execute_operations(self):
         router_id = routers.create_router(
             self._destination_openstack_client, migr_info)
 
+        src_routes = routers.get_source_routes(
```

Comment:
Please move this `get_source_routes` call inside the body of the `if` as the returned values aren't used otherwise.